### ISSUE TITLE
Hosted cryptag without sandstorm

### ISSUE THREAD

**Issue by ben-pr-p** [cryptag/cryptag#99]
Is there a way to host cryptag on a server I control, so that I could do `cryptag create -h crypt.myserver.com ...`?

That would make this very useful for storing `.env` files and updating them across my different dev environments!

**Comment by elimisteve** [comment]
Yes, but there's no auth because Sandstorm adds auth for it.  But if you add HTTP Basic Auth to Nginx or something like that, and run Go behind that, that would be secure, at which point you could do this:

After installing `Go` [(here's a script to do so)](https://github.com/PursuanceProject/install-go#install-go)

```
go get github.com/cryptag/cryptag/servers/cryptag-webserver
cryptag-webserver
```
